pkg/ingestor/parser/slsa/parser_slsa.go

```diff
@@ -19,16 +19,20 @@ import (
 	"context"
 	"errors"
 	"fmt"
+
 	"strings"
+	"time"
 
 	jsoniter "github.com/json-iterator/go"
 
-	"github.com/in-toto/in-toto-golang/in_toto"
+	slsa1 "github.com/in-toto/attestation/go/predicates/provenance/v1"
+	attestationv1 "github.com/in-toto/attestation/go/v1"
 	scommon "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/common"
 	slsa01 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.1"
 	slsa02 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
-	slsa1 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
+	smtslsa1 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
```

Comment:
Sorry, I was implying that we hardcode the value to the parser as in-toto-golang will be deprecated...